src/createRoom.ts

```diff
@@ -221,6 +221,18 @@ export default async function createRoom(opts: IOpts): Promise<string | null> {
     let roomId;
     return client.createRoom(createOpts).finally(function() {
         if (modal) modal.close();
```

Comment:
I feel like this might fire at the wrong time for this retry

Reply:
Yup, you're right--this closes the modal before the re-attempted room creation has finished. I committed a fix just now. Thanks for catching this!